docs/bin/plugin_formatter.py

```diff
@@ -338,7 +338,7 @@ def print_modules(module, category_file, deprecated, options, env, template, out
     if module in deprecated:
         modstring = modstring + DEPRECATED
 
-    category_file.write("  %s - %s <%s_module>\n" % (to_bytes(modstring), to_bytes(rst_ify(module_map[module][1])), to_bytes(modname)))
+    category_file.write("  %s - %s <%s_module>\n" % (to_text(modstring), to_text(rst_ify(module_map[module][1])), to_text(modname)))
```

Comment:
This one I'll have to track down where we open category_file.  My inclination (from wanting to be explicit about translating to byte strings) is that we should be doing:
``` python
category_file.write(b"  %s - %s <%s_module>\n" % (to_bytes(modstring), to_bytes(rst_ify(module_map[module][1])), to_bytes(modname)))
```

here but have to check if we've opened category_file in binary mode.